wiki/en/Running-a-Server.md

```diff
@@ -125,185 +148,178 @@ To register with one of the Directories built into the Jamulus Client, replace `
+For macOS, start a Terminal window and run Jamulus with the desired options like this:
 
-**Note:** The default port for the current version of Jamulus is **UDP** (not TCP) port **22124**. You will usually forward the port **22124** from outside your network to the port **22124** of the machine running the Server.
+ `/Applications/Jamulus.app/Contents/MacOS/Jamulus --serverbindip 192.168.0.100`
 
-**Note:** Your home router may change the IP address of the machine you're running your Server on. Depending on your router, you might need to give this machine a static IP address (often under DHCP settings of your router).
+**Note** Command line options will set the Server’s defaults at startup. You can override them with their corresponding GUI controls while the Server is running.
 
-#### Getting the external IP
+### The Server status icon
 
-To allow others to connect to your Server from the internet, get your external (WAN) IP address e.g. by [using Google](https://www.google.com/search?q=what+is+my+ip) and give it to them.
-You yourself should connect using the local network (LAN) address of the machine the Server is running on. If you are running a Client on the same machine as your Server, that would be `localhost` or `127.0.0.1`.
+When a Server is running in GUI mode, the operating system will show an icon in the system tray or status area that indicates whether the Server is active:  
 
+<figure><img src="{% include img/en-screenshots/server-inactive.inc %}" style="float:left; margin-right:10px;" loading="lazy" alt="Image of the Jamulus Server icon"></figure> The Server is empty
 
-#### Dynamic DNS and why you will probably need it
+<figure><img src="{% include img/en-screenshots/server-active.inc %}" style="float:left; margin-right:10px;" loading="lazy" alt="Image of the Jamulus Server icon"></figure> The Server is occupied
 
-Most domestic internet connections change their external IP address after a short period. To avoid problems with this, you might want to set up "dynamic DNS" to get a static (sub-)domain you can share with others. Please research how to do that for your specic set up. Your router might support some "dynamic DNS" providers out of the box. If this is not the case, set up a dynamic DNS client as described by the dynamic DNS provider you chose.
+---
 
 ## Backing up the Server
 
-_Note that headless Servers do not use `.ini` files. All configuration is given as command line options._
-
 {% include_relative Include-Backing-Up.md %}
 
+- Headless Servers do not use `.ini` files. All configuration is given as command line options. If you are running a server in GUI mode, it will store its configuration (after reading any command line options on start) in the `Jamululsserver.ini` file.
```

Comment:
```suggestion
- Headless Servers do not use `.ini` files. All configuration is given as command line options. If you are running a server in GUI mode, after reading any command line options on start, it will store its configuration in the `Jamululsserver.ini` file.
```